File: May-18/25-05-18/c++/01-Find-product.cpp
#include<bits/stdc++.h>

using namespace std;

const int MOD = 1e9+7;

int main()
{
    int n;
    cin >> n;
    long long ans = 1;
    
    for(int i = 0; i < n; i++)
    {
        long long val;
        cin >> val;
        ans = ((val%MOD) * (ans%MOD) )%MOD;
    }
    cout << ans << endl;
}

File: May-18/25-05-18/java/01-Find-product.java

import java.util.*;

class TestClass {
    static int MOD = (int)1e9+7;
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        long ans = 1, val = 0 ;
        for(int i = 0; i < n; i++)
        {
            val = input.nextLong();
            ans = ( (ans%MOD) * (val%MOD)) %MOD;
        }
        System.out.println(ans);

    }
}
